src/btreemap/proptests.rs

```diff
@@ -144,6 +144,22 @@ fn map_upper_bound_iter(#[strategy(pvec(0u64..u64::MAX -1 , 10..100))] keys: Vec
     });
 }
 
+#[proptest]
+fn iter_count_test(#[strategy(0..250u8)] start: u8, #[strategy(1..250u8)] size: u8) {
+    let end = std::cmp::min(start as u64 + size as u64, 255) as u8;
```

Comment:
How long does this take to run locally for you? I just ran it and it took for me 193 seconds. We shouldn't write tests that take this much time unless absolutely necessary.

You can reduce the speed of this test by simply running it for less cases. For example, you can annotate it with something like this:

```
#[proptest(cases = 10)]
```

Also, I believe this is closer to what you actually want:

```
fn iter_count_test(#[strategy(0..250u8)] start: u8, #[strategy(#start..255u8)] end: u8) {
```